### ISSUE TITLE
DateRange for markDates

### ISSUE THREAD

**Issue by SebApptien** [kekeh/angular-mydatepicker#72]
Hello, is not possible to use a dateRange for the markDates option? Like disableDateRanges.

Thanks

**Comment by kekeh** [comment]
You can mark multiple dates like an example here: https://github.com/kekeh/angular-mydatepicker/wiki/usage-of-markDates-option

**Comment by SebApptien** [comment]
Yes, I saw this possibility. But the user cannot select a period with a begin and an end (like for disableDate).
Perfect for two / three days, less userfriendly for two months.

Thank you. Have a great day.
